infoschema/tables_test.go

```diff
@@ -65,6 +65,15 @@ func (s *testSuite) TestDataForTableRowsCountField(c *C) {
 	tk.MustQuery("select table_rows from information_schema.tables where table_name='t'").Check(
 		testkit.Rows("2"))
 
+	// Test for auto increment ID.
+	tk.MustExec("drop table t")
+	tk.MustExec("create table t (c int auto_increment primary key, d int)")
+	tk.MustQuery("select auto_increment from information_schema.tables where table_name='t'").Check(
+		testkit.Rows("1"))
+	tk.MustExec("insert into t(c, d) values(1, 1)")
+	tk.MustQuery("select auto_increment from information_schema.tables where table_name='t'").Check(
+		testkit.Rows("30002"))
```

Comment:
why this return 30002?

Reply:
@zimulala Allocate a batch of 30000 ` AUTO_INCREMENT `?

Reply:
@XuHuaiyu 
Because the next global auto ID is 30002.
We can do some operations as follows:
TiDB 1
```
tidb> create table t(`id` int(11) NOT NULL AUTO_INCREMENT,`c` int(11) DEFAULT NULL, PRIMARY KEY (`id`));
Query OK, 0 rows affected (0.13 sec)

tidb> insert into t values(1,1);
Query OK, 1 row affected (0.14 sec)

tidb> insert into t values(10000,1);
Query OK, 1 row affected (0.02 sec)

tidb> select * from t;
+-------+------+
| id    | c    |
+-------+------+
|     1 |    1 |
| 10000 |    1 |
+-------+------+
2 rows in set (0.01 sec)
```
TiDB 2
Then we do the operation as follows:
```
tidb> insert into t values();
Query OK, 1 row affected (0.13 sec)

tidb> select * from t;
+-------+------+
| id    | c    |
+-------+------+
|     1 |    1 |
| 10000 |    1 |
| 30002 | NULL |
+-------+------+
3 rows in set (0.01 sec)
```

Reply:
@ciscoxll 
Yes.